go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/google/gofuzz v0.0.0-20161122191042-44d81051d367
 	github.com/googleapis/gnostic v0.0.0-20170426233943-68f4ded48ba9
 	github.com/json-iterator/go v0.0.0-20180612202835-f2b4162afba3
+	github.com/kr/pretty v0.1.0 // indirect
```

Comment:
@liggitt is it supposed to add these?

Reply:
running `go mod tidy` should clean it up. `go get` will speculatively add things it's unsure about, and `go mod tidy` will clean them up once it verifies all os/arch/build tag combinations don't need it

Reply:
I guess not, it looks like it succeeds the tests when i remove this line

Reply:
Not sure how this got here

Reply:
When I ran go mod tidy, it didn't remove this, so I just removed it manually.